### PR TITLE
Update to React Native 0.70.13.

### PR DESCRIPTION
**Pull request by brandonpage**
This fixes an [Xcode 15 compilation error](https://github.com/facebook/react-native/issues/37748).

### CODE REVIEW

File: iosTests/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
         "@react-native-community/eslint-config": "^2.0.0",
         "@types/jest": "^26.0.24",
         "@types/react": "^18.0.21",
-        "@types/react-native": "^0.70.6",
+        "@types/react-native": "^0.70.13",
```
Comment (wmathurin) [inline iosTests/package.json:24]: I don't see a 0.70.13 for @types/react-native. See [here](https://www.npmjs.com/package/@types/react-native?activeTab=versions).

File: package.json [hunk 2]
```diff
@@ -64,7 +64,7 @@
   ],
   "devDependencies": {
     "@types/chai": "^4.2.14",
-    "@types/react-native": "^0.70.6",
+    "@types/react-native": "^0.70.13",
```
Comment (wmathurin) [inline package.json:67]: I don't see a 0.70.13 for @types/react-native. See [here](https://www.npmjs.com/package/@types/react-native?activeTab=versions).

File: iosTests/package.json (hunk 1 above)
Comment (brandonpage) [reply iosTests/package.json:24]: 😬 [It exists as a release](https://github.com/facebook/react-native/releases/tag/v0.70.13) and worked in my testing.

Comment (brandonpage) [reply iosTests/package.json:24]: @wmathurin [It's there](https://www.npmjs.com/package/@types/react-native/v/0.70.13).  The releases are sorted by recent downloads, not by tag name (they just happen to line up).  If you scroll down you will find `0.70.13`.

Comment (wmathurin) [reply iosTests/package.json:24]: You are right !! It's sorted in chronological order. I should have done a "search in page".

**Review by wmathurin — APPROVED** [review]

**Pull request merged**